ghcide/src/Development/IDE/Plugin/Completions/Logic.hs

```diff
@@ -890,3 +929,33 @@ mergeListsBy cmp all_lists = merge_lists all_lists
         []     -> []
         [xs]   -> xs
         lists' -> merge_lists lists'
+
+
+getCompletionPrefix :: (Monad m) => Position -> VFS.VirtualFile -> m (Maybe PosPrefixInfo)
```

Comment:
Just requiring a `Monad` constraint is a bit suspicious: do you need that at all, can you not just return `Maybe PosPrefixInfo`?

Reply:
Yep, not sure why it was like that